.github/workflows/ci-workflow.yml

```diff
@@ -10,10 +10,6 @@ on:
 
 jobs:
 
-  guideline_enforcer:
```

Comment:
Why remove this ?

Reply:
Duplicate of a dedicated job:
https://github.com/LedgerHQ/app-stellar/blob/develop/.github/workflows/guidelines_enforcer.yml